docs/developers/developers-setup.md

```diff
@@ -177,8 +177,7 @@ To run Diffuse:
 diffuse
 ```
 
-_Note: The `diffuse` command can be used to launch Diffuse as a native Mac app_
-_that is installed into `/Applications/Diffuse.app`._
+_Note: The `diffuse` command can be used to launch Diffuse as a native Mac app that is installed into `/Applications/Diffuse.app`. To pass command-line arguments, run `diffuse_impl`. _
```

Comment:
Command line args can actualy be passed to `diffuse` too (otherwise `diffuse` would not had been compatible with `git difftool`) - it's just that `open` seems to swollow all output so you don't see it.

Maybe we can fix that? Running `diffuse_impl` directly is not ideal because [Diffuse will not get its icon, etc](
https://github.com/MightyCreak/diffuse/blob/master/src/diffuse/mac-os-app/diffuse-mac.in#L5). If we can find a fix for `diffuse` or `open`, that would be much better than recommending users to call the internal impl.

Reply:
I'll extract this to a separate PR so that we don't forget.